sssom/util.py

```diff
@@ -267,13 +271,20 @@ def remove_unmatched(df: pd.DataFrame) -> pd.DataFrame:
     """Remove rows where no match is found.
 
     TODO: https://github.com/OBOFoundry/SSSOM/issues/28
-    :param df:
-    :return:
+    :param df: Pandas DataFrame
+    :return: Pandas DataFrame with 'PREDICATE_ID' not 'noMatch'.
     """
     return df[df[PREDICATE_ID] != "noMatch"]
 
 
 def create_entity(row, eid: str, mappings: Dict[str, Any]) -> Entity:
+    """Create an Entity object.
+
+    :param row: Row - [UNUSED]
```

Comment:
why not just remove it?

Reply:
@matentzn

Reply:
as discussed in call, just remove all mentions of row. MAke sure all calls to create_entity use parameter names (create_entity(eid="a"), not create_entity(eid))

Reply:
looks like this is done